File: scratch/parse_excel.js
import XLSX from "xlsx";
import fs from "fs";

const workbook = XLSX.readFile("India_Top_500_Tourist_Destinations.xlsx");
console.log("Sheet Names:", workbook.SheetNames);

const sheetName = workbook.SheetNames[0];
const sheet = workbook.Sheets[sheetName];

const data = XLSX.utils.sheet_to_json(sheet);
console.log("Total Records:", data.length);
console.log("Headers:", Object.keys(data[0] || {}));
console.log("\nSample 5 Records:");
console.log(data.slice(0, 5));

File: scratch/parse_excel.py
import openpyxl
import json

wb = openpyxl.load_workbook('India_Top_500_Tourist_Destinations.xlsx')
print("Sheet names:", wb.sheetnames)

sheet = wb.active
rows = list(sheet.iter_rows(values_only=True))

if rows:
    headers = rows[0]
    print("Headers:", headers)
    print("Total rows:", len(rows) - 1)
    print("\nSample Rows:")
    for row in rows[1:6]:
        print(row)
